projects/components/src/quick-search/quick-search.component.spec.ts

```diff
@@ -200,67 +234,70 @@ describe('QuickSearchComponent', () => {
             const searchHandlerSpy = spyOn(this.quickSearchData.simpleProvider, 'searchHandler').and.callThrough();
             this.finder.hostComponent.spotlightOpen = true;
             this.finder.detectChanges();
-            this.quickSearch.searchInputValue = 'c';
+            this.quickSearch.getInput().type('c');
             expect(searchHandlerSpy).toHaveBeenCalledWith('c');
-            expect(this.quickSearch.searchResults).toEqual(['copy', 'create']);
+            expect(this.quickSearch.getSearchResultItems().map((item) => item.text())).toEqual(['copy', 'create']);
         });
 
         it('displays a loading indicator when the result is a promise', function (this: Test): void {
             this.quickSearchData.spotlightSearchService.registerProvider(this.quickSearchData.asyncProvider);
             this.finder.hostComponent.spotlightOpen = true;
             this.finder.detectChanges();
-            this.quickSearch.searchInputValue = 'c';
-            expect(this.quickSearch.isLoading).toBeTruthy('There should be a loading indicator in the second section');
+            this.quickSearch.getInput().type('c');
+            expect(this.quickSearch.getSpinner().length()).not.toBe(
```

Comment:
NIT: can we change this to `expect(this.quickSearch.getSpinner().length()).toBe(1, ...)` or whatever the correct number is. Just find it easier to read